delfin/task_manager/scheduler/schedulers/telemetry/failed_performance_collection_handler.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2021 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import six
+from oslo_config import cfg
+from oslo_log import log
+
+from delfin import db
+from delfin import exception
+from delfin.common.constants import TelemetryJobStatus
+from delfin.db.sqlalchemy.models import FailedTask
+from delfin.db.sqlalchemy.models import Task
+from delfin.i18n import _
+from delfin.task_manager import rpcapi as task_rpcapi
+from delfin.task_manager.scheduler import scheduler
+from delfin.task_manager.tasks.telemetry import PerformanceCollectionTask
+
+LOG = log.getLogger(__name__)
+CONF = cfg.CONF
+
+
+class FailedPerformanceCollectionHandler(object):
+    def __init__(self, ctx, filed_task_id, storage_id, args, job_id,
+                 start_time, end_time):
+        self.ctx = ctx
+        self.filed_task_id = filed_task_id
+        self.retry_count = 0
+        self.storage_id = storage_id
+        self.job_id = job_id
+        self.args = args
+        self.start_time = start_time
+        self.end_time = end_time
+        self.task_rpcapi = task_rpcapi.TaskAPI()
+        self.scheduler_instance = scheduler.Scheduler.get_instance()
+        self.result = TelemetryJobStatus.FAILED_JOB_STATUS_INIT
+
+    @classmethod
```

Comment:
Why use classmethod instead of staticmethod

Reply:
Updated as per suggestion. Thanks